Fixing formatting in EBS API Section (#134)
C2Devel/terraform-provider-croccloud: docs/resources/ebs_snapshot_import.md

Reviewer: S3 -- маркетинговый термин
```suggestion
  Imports a disk image from object storage as a snapshot.
```

Reviewer: ```suggestion
Imports a disk image from object storage as a snapshot.
```

Reviewer: тут наверно корректнее так
```suggestion
* `s3_key` - The name of the disk image file.
```

Reviewer: см. стайлгайд, аналогично в других местах
```suggestion
* `id` - The ID of the snapshot (e.g., `snap-12345678`).
```

Reviewer: видимо имелся ввиду disk drive, но не используем такую терминологию (м.б. стоит уточнить, почему здесь атрибут называется volume, когда в других местах disk)
мигает запятая перед in GiB по всей доке. Надо зафиксировать, как пишем. По грамматике д.б. с зпт
```suggestion
* `volume_size` - The size of the volume, in GiB.
```

Author: Покопал информацию. Запятая здесь не нужна, ни по правилам, ни по MS Style Guide, ни по некоторым прочим стайлгайдам. Она здесь может быть нужна по правилам русского языка (но и то более уместны скобки, которых мы избегаем)

Reviewer: ОК, давай тогда будем убирать

Author: Убрал везде

layout: "aws"
page_title: "aws_ebs_snapshot_import"
description: |-
  Imports a disk image from object storage as a snapshot.
---

[default-tags]: https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block

> [timeouts]: https://www.terraform.io/docs/configuration/blocks/resources/syntax.html#operation-timeouts

Reviewer 2: Старая ссылка

# Resource: aws_ebs_snapshot_import

> Imports a disk image from object storage as a snapshot.

Reviewer 2: Возможно нужен артикль перед `object storage`

Author: Мы тут, насколько я понимаю, импортируем имадж не из какого-то хранилища, а из объектного хранилища вообще как сущности) поэтому артикль не надо @PixarV

}
```

## Argument reference

The following arguments are required:

> * `disk_container` - (Required, Editable, [Block](#disk_container)) Information about the disk container.

Reviewer 2: ```suggestion
* `disk_container` - (Required, Forces new resource, [Block](#disk_container)) Information about the disk container.
```

> * `disk_container` - (Required) Information about the disk container. The structure of this block is [described below](#disk_container).

Reviewer: а здесь надо ссылку на блок сделать, как в VPC делаем, чтобы потом не переделывать

The following arguments are optional:

> * `description` - (Optional, String) The description string for the import snapshot task.

Reviewer: если "задача импорта снапшота", то 
хотя aws вроде пишет так, как написано
```suggestion
* `description` - (Optional, String) The description string for the snapshot import task.
```

* `tags` - (Optional, Map of strings) Key-value pairs to assign to the snapshot. If the [`default_tags` configuration block][default-tags] is used within a provider configuration, the tags with matching keys will overwrite those defined at the provider level.

### disk_container

The following arguments are required:

* `format` - (Required, Forces new resource, String) The format of the disk image being imported.

>     * _Valid values:_  `RAW`, `VHD`, `VMDK`

Reviewer 2: Еще `QCOW2`

>     * _Valid values:_  `RAW`, `VHD`, `VMDK`.

Reviewer: ```suggestion
    * _Valid values:_  `RAW`, `VHD`, `VMDK`
```

> * `user_bucket` - (Required, Editable, [Block](#user_bucket)) The bucket for the disk image.

Reviewer 2: ```suggestion
* `user_bucket` - (Required, Forces new resource, [Block](#user_bucket)) The bucket for the disk image.
```

The following arguments are optional:

* `description` - (Optional, String) The description of the disk image being imported.

#### user_bucket

The following arguments are required:

* `s3_bucket` - (Required, Forces new resource, String) The name of the bucket where the disk image is located.
* `s3_key` - (Required, Forces new resource, String) The name of the disk image file.

## Attribute reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

* `arn` - (String) The Amazon Resource Name (ARN) of the snapshot.
* `id` - (String) The ID of the snapshot.
    * _Example:_ `snap-12345678`
* `owner_alias` - (String) The alias of the snapshot owner.
* `owner_id` - (String) The ID of the project.
* `volume_size` - (Integer) The size of the volume in GiB.

> * `tags_all` - (Map of tags) Key-value pairs assigned to the snapshot, including any tags inherited from the [`default_tags` configuration block][default-tags] if used within a provider configuration.

Reviewer 2: ```suggestion
* `tags_all` - (Map of strings) Key-value pairs assigned to the snapshot, including any tags inherited from the [`default_tags` configuration block][default-tags] if used within a provider configuration.
```

## Argument Reference

The following arguments are required:

* `disk_container` - (Required) Information about the disk container. The structure of this block is [described below](#disk_container-configuration-block).

The following arguments are optional:

* `description` - (Optional) (String) The description string for the import snapshot task.
* `tags` - (Optional) (Map of tags) Key-value map of tags to assign to the snapshot.
  If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

### disk_container

The following arguments are required:

* `format` - (Required, Forces new resource) (String) The format of the disk image being imported.
    * _Valid values:_  `RAW`, `VHD`, `VMDK`.

> * `user_bucket` - (Required) The bucket for the disk image.

Reviewer 2: Тут должна быть ссылка на блок ниже

* `description` - (Optional) The description of the disk image being imported.
* `format` - (Required) The format of the disk image being imported. One of `VHD`, `VMDK`, `QCOW2` or `RAW`.
* `user_bucket` - (Required) The S3 bucket for the disk image.

> 

Reviewer 2: Эти строчки можно удалить, тк параметры перенесены

The following arguments are supported:

> * `disk_container` - (Required) Information about the disk container. Detailed below.

Reviewer: надо ссылку на блок дать (см. оформление в других местах)

* `description` - (Optional) The description string for the import snapshot task.
* `tags` - (Optional) Map of tags to assign to the snapshot.
  If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

### disk_container Configuration Block

> * `format` - (Required) The format of the disk image being imported.

Reviewer: порядок аргументов -- сначала необходимые, потом опциональные.
и те и другие должны быть отсортированы по алфавиту

Author: Так и делал, по стайлгайду)

>     * _Valid values:_ `VHD`, `VMDK` or `RAW`.

Reviewer: точку в конце ставим только после описания параметра
or допустимо, но лучше перечисление через запятую
и я бы уточнил у Риты, м.б. как у нас в доке принято в качестве разделителя использовать |
```suggestion
    * _Valid values:_ `VHD`, `VMDK`, `RAW`
```

> * `user_bucket` - (Required) The S3 bucket for the disk image.

Reviewer: формально у нас объектное хранилище (S3 это амазоновский бренд, его используем в маркетинговых текстах)

```suggestion
* `user_bucket` - (Required) The bucket for the disk image.
```

Reviewer: здесь тоже не изменено и не прокомментировано

Reviewer: не поправлено

The following arguments are supported:

* `disk_container` - (Required) Information about the disk container. The structure of this block is [described below](#disk_container-configuration-block).
* `description` - (Optional) The description string for the import snapshot task.
* `tags` - (Optional) Map of tags to assign to the snapshot.
  If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.

> ### disk_container Configuration Block

Reviewer 2: ```suggestion
### disk_container
```

В других местах выносим только имя блока в заголовок. Нашла такой вариант формления только у eks ресурсов, там тоже можно поправить

* `format` - (Required) The format of the disk image being imported.

>     * _Valid values:_ `VHD`, `VMDK`, `RAW`.

Reviewer: Проверь плиз, что в других подобных местах нет точки
```suggestion
    * _Valid values:_ `VHD`, `VMDK`, `RAW`
```

In addition to all arguments above, the following attributes are exported:

* `arn` - The Amazon Resource Name (ARN) of the EBS snapshot.

> * `id` - The ID of the snapshot (e.g., `snap-12345678`).

Reviewer: а здесь не сделано форматирование для Example (надо было во всех местах поменять в разделе EBS)

* `owner_alias` - The alias of the EBS snapshot owner.

> * `owner_id` - The project ID.

Reviewer: Написание надо обязательно обсудить с Ритой, ее просьба везде писать одинаково (The ID of the project)

Author: Обсудили на троих, будем делать The X of the Y

The `timeouts` block allows you to specify [timeouts] for certain actions:

- `create` - (Default `60 minutes`) Used for importing the snapshot.
- `delete` - (Default `10 minutes`) Used for deleting the snapshot.

## Import

> To import a snapshot, use the [aws_ebs_snapshot](ebs_snapshot.md#import) resource.

Reviewer: и тут тот же вопрос,  the или a (выше the)

Author: Тут всё-таки, на мой взгляд, `a`, потому что тут речь об импорте рандомного снапшота, не какого-то конкретного @DiGa66

The `timeouts` block allows you to specify [timeouts] for certain actions:

- `create` - (Default `60 minutes`) Used for importing the snapshot.
- `delete` - (Default `10 minutes`) Used for deleting the snapshot.

## Import

> Import of the snapshots is not currently supported.

Reviewer 2: Не совсем корректно. Сами снапшоты импортировать можно, для этого используется ресурс aws_snapshot. Не поддерживается импорт ресурса "импорт снапшота". Надо подумать как это тут сформулировать

Author: Сделал ссылку на ebs_snapshot
